components/l10n/browser/default_locale_util_linux.cc

```diff
@@ -7,13 +7,13 @@
 
 #include <cstdlib>
 #include <optional>
+#include <string>
 
 namespace brave_l10n {
 
 std::optional<std::string> MaybeGetDefaultLocaleString() {
   // LC_ALL should always override the LANG variable, whether it is set or not.
-  char const* language = language = std::getenv("LC_ALL");
-
+  const char* language = std::getenv("LC_ALL");
```

Comment:
Is there a particular reasons you didn't use `base/environment.h`?

Reply:
Thanks, changed to use base/environment.h